monitor/redis/watcher.go

```diff
@@ -70,20 +70,35 @@ func (w *Watcher) monitor(ctx context.Context, ch chan<- []*change.Change) {
 	}
 }
 
+func toPtr(s string) *string {
+	return &s
+}
+
 func (w *Watcher) getValues(ctx context.Context, ch chan<- []*change.Change) {
-	values, err := w.client.MGet(ctx, w.keys...).Result()
```

Comment:
deleted keys would get `nil` values as you can see from the lib unit tests:

https://github.com/go-redis/redis/blob/master/commands_test.go#L1165-L1173

It means that deletion operations would not be pickedup by the service.